editor/gui/scene_tree_editor.cpp

```diff
@@ -511,24 +512,35 @@ void SceneTreeEditor::_queue_update_node_tooltip(Node *p_node, TreeItem *p_item)
 }
 
 void SceneTreeEditor::_update_node_tooltip(Node *p_node, TreeItem *p_item) {
+	static StringBuilder tooltip;
```

Comment:
Is there any tangible benefit in making it static instead of creating new StringBuilder every time? (like in every other usage of this class)

Reply:
Without this static `StringBuilder` will not be reused so the performance is simillar to third graph (no improvement). Replace String `+` with `StringBuilder` has no performance benifit, performance benifit comes from reducing allocations.

Reply:
Maybe add a comment then.